test/e2e/performanceprofile/functests/1_performance/performance.go

```diff
@@ -1018,6 +1019,52 @@ var _ = Describe("[rfe_id:27368][performance]", func() {
 			})
 		})
 	})
+
+	Context("Offlined CPU API", func() {
+		It("should set offline cpus after deploy PAO", func() {
```

Comment:
let's add a label to make it possible to skip (or to focus on) this very potentially disruptive test, something like` [disruptive]` or something else